lib/controllers/platform-controller.ts

```diff
@@ -143,6 +143,20 @@ export class PlatformController implements IPlatformController {
 		const hasPlatformDirectory = this.$fs.exists(
 			path.join(projectData.platformsDir, platformName)
 		);
+
+		if (hasPlatformDirectory) {
+			const platformDirectoryItemCount = this.$fs.readDirectory(
+				path.join(projectData.platformsDir, platformName)
+			).length;
+
+			if (platformDirectoryItemCount <= 5) {
```

Comment:
```suggestion
			// 5 is a magic number to approximate a valid platform folder
			// any valid platform should contain at least 5 files
			// we choose 5 to avoid false-positives due to system files like .DS_Store etc.
			if (platformDirectoryItemCount <= 5) {
```